mne/viz/_figure.py

```diff
@@ -2295,6 +2074,88 @@ def _toggle_vline(self, visible):
         self.mne.vline_visible = visible
         self.canvas.draw_idle()
 
+    def close_all(self):
+        plt.close('all')
+
+    # workaround: plt.close() doesn't spawn close_event on Agg backend
+    # (check MPL github issue #18609; scheduled to be fixed by MPL 3.4)
+    def close_event(self, target=None):
+        """Force calling of the MPL figure close event."""
+        target = target or self
+        try:
+            target.canvas.close_event()
+        except ValueError:  # old mpl with Qt
+            pass  # pragma: no cover
+
+    def get_n_windows(self):
+        return len(plt.get_fignums())
+
+    def press_key(self, key, target=None):
+        target = target or self
+        target.canvas.key_press_event(key)
+
+    def fake_click(self, point, target=None, ax=None,
+                   xform='ax', button=1, kind='press'):
+        """Fake a click at a relative point within axes."""
+        target = target or self
+        ax = ax or self.mne.ax_main
+        if xform == 'ax':
+            x, y = ax.transAxes.transform_point(point)
+        elif xform == 'data':
+            x, y = ax.transData.transform_point(point)
+        else:
+            assert xform == 'pix'
+            x, y = point
+        if kind == 'press':
+            func = partial(target.canvas.button_press_event, x=x, y=y,
+                           button=button)
+        elif kind == 'release':
+            func = partial(target.canvas.button_release_event, x=x, y=y,
+                           button=button)
+        elif kind == 'motion':
+            func = partial(target.canvas.motion_notify_event, x=x, y=y)
+        else:
+            raise RuntimeError(f'{kind} is no suitabl kind for _fake_click')
+
+        func(guiEvent=None)
```

Comment:
would it work to have the figure method be a thin wrapper to a utility function like the existing `_fake_click` function?  I think we use fake clicks in testing for other plots besides just the data browser... might be more DRY

Reply:
Yes thats right, I changed the signature a bit by adding defaults to target(former fig) and ax which results in reduced calls from the tests. But that can be done better by wrapping `_fake_click `from `mne.viz.utils`, thanks for the hint.